src/test/java/io/outfoxx/swiftpoet/test/FunctionSpecTests.kt

```diff
@@ -796,4 +797,48 @@ class FunctionSpecTests {
     assertThat(testFuncBlder.signature.parameters, hasItems(ParameterSpec.builder("a", STRING).build()))
     assertThat(testFuncBlder.body.formatParts, hasItems("val;\n"))
   }
+
+  @Test
+  @DisplayName("Generate property with setter and getter")
```

Comment:
Hi @comm1x, just verifying, this test fails without your change correct?

Reply:
Yes.